lib/export.js

```diff
@@ -170,4 +171,11 @@ function relativeImportPath(fromIdentifier, toIdentifier) {
   return [...fromPath, ...toNS, toIdentifier.name].join('/');
```

Comment:
Since this is building an import path, which ends in a class name, we probably need to sanitize `toIdentifier.name`.